java-frontend/src/main/java/org/sonar/java/classpath/ClasspathProperties.java

```diff
@@ -37,25 +38,30 @@ private ClasspathProperties() {
 
   public static List<PropertyDefinition> getProperties() {
     List<PropertyDefinition> extensions = new ArrayList<>();
+    extensions.add(PropertyDefinition.builder(SONAR_JAVA_JDK_HOME)
+      .description("Path to jdk directory used by the project under analysis.")
+      .hidden()
+      .build()
+    );
     extensions.add(PropertyDefinition.builder(SONAR_JAVA_BINARIES)
-            .description("Comma-separated paths to directories containing the binary files (directories with class files).")
-            .hidden()
-            .build()
+      .description("Comma-separated paths to directories containing the binary files (directories with class files).")
```

Comment:
Unrelated, but would be great to declare those properties as multi-valued

Reply:
since they are hidden anyway, i'm not sure it would change things, but indeed.